features/user_associated_with_multiple_accounts.md

```diff
@@ -0,0 +1,27 @@
+# User associated with multiple accounts
+
+A user can be associated with multiple ImageKit accounts using the same email address. A previously registered user can be added to a different account. Read how to add a user to an account [here](./user-access-management/README.md).
+
+The login flow for a multi-account user is as follows:
+
+You will be asked to put in your email and password (any valid password for one of the accounts) at the login screen.
```

Comment:
At the login screen, you will be asked to enter your email address and password (any valid password for one of the accounts).